14/umbraco-cms/fundamentals/backoffice/sections.md

```diff
@@ -60,35 +62,38 @@ The Settings section consists of:
 * Extension Insights
 * Webhooks
 
-The **Settings** section of the Umbraco backoffice has its own set of default dashboards. For more information, see the [Settings Dashboards](settings-dashboards.md) article.
+The **Settings** section in the Umbraco backoffice has its own set of default dashboards. For more information, see the [Settings Dashboards](settings-dashboards.md) article.
 
 ## Packages
 
 In this section, you can browse and install packages into your Umbraco solution. You can also get an overview of all the installed packages as well as uninstall packages you no longer need.
 
 ## Users
 
-Manage, create, and customize Backoffice users and user groups.
+The Users section allows administrators to manage user accounts, assign permissions, set user roles, and monitor user activity within the backoffice. It provides control over who can access and modify content, media, and settings in the CMS.
 
 ## Members
 
-Manage, create, and customize members and member groups.
+The Members section allows to create and manage member profiles, set up member groups, and control Member's access to restricted content on the website.
 
-## Forms
+## Translation
 
-You can install Umbraco Forms directly from the Backoffice by clicking the install button. Once installed, this section is where you create and manage your forms.
+The Translation section is where you create and manage Dictionary Items. By managing these dictionary items, you can ensure consistent and efficient content translation and maintenance across different languages.
 
-## Translation
+## Add-On Sections
 
-This is the section where you create and manage your dictionary items.
+You can enhance Umbraco's functionality with plugins and extensions tailored to specific needs, expanding capabilities beyond core features. Currently, Umbraco supports add-ons like Forms, Deploy, Workflow, Commerce, and UI Builder.
```

Comment:
How about a link to the Marketplace here?

Reply:
Added a link to the marketplace